examples/common-switches.mk

```diff
@@ -85,7 +85,6 @@ THREAD_VERSION            ?= 1.1
 TIME_SYNC                 ?= 0
 UDP_FORWARD               ?= 0
 RCP_RESTORATION_MAX_COUNT ?= 0
-S2S                       ?= 0
```

Comment:
BTW, why do we remove these?

Reply:
There is a duplicated conf `SSED_SSED_LINK` at line 83.